hardpy/pytest_hardpy/pytest_call.py

```diff
@@ -602,22 +619,22 @@ def run_dialog_box(dialog_box_data: DialogBox) -> Any:  # noqa: ANN401
 
     Returns:
         Any: An object containing the user's response.
-
         The type of the return value depends on the widget type:
-
         - BASE: bool.
         - TEXT_INPUT: str.
         - NUMERIC_INPUT: float.
         - RADIOBUTTON: str.
         - CHECKBOX: list[str].
         - MULTISTEP: bool.
+        Any: The return value of the dialog box.
```

Comment:
`Any` description is at the beginning of the documentation line.

```suggestion
        - Pass/Fail widget: PassFailDialog.
```